### DISCUSSION TITLE
Task 1: Lab 02.03 Extract data

### DISCUSSION THREAD

**Discussion by jaayg23** [rramosp/ai4eng.v1#123] [02 - PYTHON]
Buenas noches agradecería que me ayudaran con el siguiente error en el task.
<img width="1389" alt="Captura de Pantalla 2022-05-07 a la(s) 7 00 57 p  m" src="https://user-images.githubusercontent.com/73083558/167276038-d34dc9f5-3e08-4230-a4bf-0e6354733642.png">
Adjunto mi posible solución para encontrar el error.

`def select_items(df):
    # make sure to make a copy in case you modify the original df
    df = df.copy()
    
    if "margin" in d.columns:
      df = df[(df.price>100)|(df.margin>10)]
    else:
      df = df[(df.price>100)]
    
    result = df.index.tolist()
    
    return result`

**Comment by rramosp** [comment]
jacobo, tienes que fijarte bien en el error que te da y el nombre de
variable al que se refiere.

On Sat, May 7, 2022 at 7:03 PM Jacobo Ayala Giraldo <
***@***.***> wrote:

> Buenas noches agradecería que me ayudaran con el siguiente error en el
> task.
> [image: Captura de Pantalla 2022-05-07 a la(s) 7 00 57 p  m]
> <https://user-images.githubusercontent.com/73083558/167276038-d34dc9f5-3e08-4230-a4bf-0e6354733642.png>
> Adjunto mi posible solución para encontrar el error.
>
> `def select_items(df):
> # make sure to make a copy in case you modify the original df
> df = df.copy()
>
> if "margin" in d.columns:
>
>   df = df[(df.price>100)|(df.margin>10)]
>
> else:
>
>   df = df[(df.price>100)]
>
>
>
> result = df.index.tolist()
>
>
>
> return result`
>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/123>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZILCBECRCN36V4GQRH3VI4AGBANCNFSM5VLDD3VQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
